### PR TITLE
Dev -> Master (Adding OpenAMP Logo / member logos #7)

### PR DESCRIPTION
**Pull request by kylekirkby**
Dev -> Master (Adding OpenAMP Logo / member logos #7)

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-title: OpenAMPProject
+title: OpenAMP Project
 description: >-
-  ArmNN and Arm Compute Library
+  OpenAMP (Open Asymmetric Multi-Processing) seeks to standardize the interactions between operating environments in a heterogeneous embedded system through open source solutions for Asymmetric MultiProcessing (AMP).
 url: "https://openampproject.org"
```
Comment (pcolmer) [inline _config.yml]: https://www.openampproject.org ?

File: _data/settings.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ newsletter:
 # Blog tags
 tags_enabled: true
 # Jekyll social media site settings. Github, LinkedIn, Google+, Facebook and Youtube
-github_username: ArmNNWebsite
+github_username: OpenAMP
```
Comment (pcolmer) [inline _data/settings.yml]: Sure? Not OpenAMPProject?

**Comment by kylekirkby** [comment]
> Hi Kyle, looks really nice!
> 
> Is it OK to leave the member logos off the index page? I worry individual contributors would get scared off if they see that it's just big companies on the front page. I rather they see the logos in the About where there is more context, like a statement welcoming all contributors.
> 
> Also it looks like the description on the index page is causing the geometric pattern band to fall below the black transparent band, making the white text hard to read:
> image. That was in Firefox Quantum 69.0 (64-bit) for Ubuntu canonical-1.0.
> 
> Thanks for putting this together :)
> -Nathalie

Just pushed a couple of commits to fix the above! Thanks Nathalie!

Kyle

File: _config.yml (hunk 1 above)
Comment (kylekirkby) [reply _config.yml]: [https://mlplatform.org](https://mlplatform.org) - Is this site a special case then?

Comment (pcolmer) [reply _config.yml]: I'd need to check with @morancj to be sure but I thought that if you leave "www." off, it redirects to "www.", so it is better to have the correct FQDN there.

A bit like always having the trailing slash as well - it eliminates a round-trip between the browser and the server.

Comment (kylekirkby) [reply _config.yml]: Sure - thanks @pcolmer ! I'll update this!

Comment (morancj) [reply _config.yml]: @kylekirkby as of the new static site deployment, the apex record resolves to `www.`:
```
➜ curl -IL http://openampproject.org/
HTTP/1.1 301 Moved PermanentlyContent-Length: 0
Connection: keep-alive
Date: Tue, 17 Sep 2019 14:30:12 GMT
Location: https://www.openampproject.org/
Server: AmazonS3
X-Cache: Hit from cloudfront
Via: 1.1 42322007e22ea6a235ae829b1f254f98.cloudfront.net (CloudFront)
X-Amz-Cf-Pop: MAN50-C1
X-Amz-Cf-Id: U7JoXNq3zLk3ElIFfTFaK07SgZ3DCHRTfAE1dg4WTtLZmRRB7nYXQA==
Age: 1051

HTTP/2 200 
content-type: text/html
content-length: 6750
date: Tue, 17 Sep 2019 14:30:12 GMT
cache-control: public, max-age=86400
last-modified: Tue, 17 Sep 2019 13:24:12 GMT
etag: "8153ebf309439eaebb99e44a3185e143"
server: AmazonS3
strict-transport-security: max-age=31557601; preload
x-content-type-options: nosniff
vary: Accept-Encoding
x-cache: Hit from cloudfront
via: 1.1 340acc109468df693ec024a66dbb0f1e.cloudfront.net (CloudFront)
x-amz-cf-pop: LHR61-C2
x-amz-cf-id: 8v1cTuOGjFXjcl8xDEyZXnmola4EbuCkrcl6HNNkbv5oDxStLdaQaQ==
age: 1051
```
Similar if you start with `https://openampproject.org/`. @pcolmer Agree re. trailing slash. :)

**Comment by morancj** [comment]
> That was in Firefox Quantum 69.0 (64-bit) for Ubuntu canonical-1.0.

FYI, for anyone who cares, see also [https://bugs.launchpad.net/ubuntu/+source/firefox/+bug/418203] :smile: 
Looks like:

> 69.0 (64-bit)
> Mozilla Firefox for Arch Linux
> archlinux - 1.0

on this distribution.

**Comment by nathalie-ckc** [comment]
> > Hi Kyle, looks really nice!
> > Is it OK to leave the member logos off the index page? I worry individual contributors would get scared off if they see that it's just big companies on the front page. I rather they see the logos in the About where there is more context, like a statement welcoming all contributors.
> > Also it looks like the description on the index page is causing the geometric pattern band to fall below the black transparent band, making the white text hard to read:
> > image. That was in Firefox Quantum 69.0 (64-bit) for Ubuntu canonical-1.0.
> > Thanks for putting this together :)
> > -Nathalie
> 
> Just pushed a couple of commits to fix the above! Thanks Nathalie!
> 
> Kyle

Looks great. Thanks, Kyle!
-Nathalie

**Comment by nathalie-ckc** [comment]
> > That was in Firefox Quantum 69.0 (64-bit) for Ubuntu canonical-1.0.
> 
> FYI, for anyone who cares, see also [https://bugs.launchpad.net/ubuntu/+source/firefox/+bug/418203] 😄
> Looks like:
> 
> > 69.0 (64-bit)
> > Mozilla Firefox for Arch Linux
> > archlinux - 1.0
> 
> on this distribution.

I was thinking Ubuntu canonical-1.0 seemed a little weird. Impressive that it has dragged on since 2009!

**Pull request merged**